Rakefile

```diff
@@ -64,11 +64,20 @@ rescue LoadError
   end
 end
 
-desc "Run rubocop"
-task(:rubocop) do
-  sh "util/rubocop"
+namespace :rubocop do
+  desc "Run rubocop for RubyGems"
+  task(:rubygems) do
+    sh "util/rubocop"
+  end
+
+  desc "Run rubocop for Bundler"
+  task(:bundler) do
+    sh "util/rubocop --ignore_parent_exclusion --config bundler/.rubocop.yml bundler"
```

Comment:
So, is the `--ignore_parent_exclusion` flag actually needed? I think we no longer need it since we are running `rubocop` from the root folder.

Reply:
Let's attempt without it.